library/orientdb

```diff
@@ -1,17 +1,17 @@
 
 Maintainers: Luigi Dell'Aquila <l.dellaquila@orientdb.com> (@luigidellaquila)
 
-Tags: 3.2.26, 3.2, latest
+Tags: 3.2.27, 3.2, latest
 Architectures: amd64, arm32v7, arm64v8
 GitRepo: https://github.com/orientechnologies/orientdb-docker.git
-GitCommit: 1cf7837740ef309efc58759b0a33a3fe8c2c5278
-Directory: release/3.2.x/3.2.26
+GitCommit: 7c3a945a49494db55de300c0193da8d9dd0f51c1
+Directory: release/3.2.x/3.2.27
 
-Tags: 3.2.26-tp3, 3.2-tp3
+Tags: 3.2.27-tp3, 3.2-tp3
 Architectures: amd64, arm32v7, arm64v8
 GitRepo: https://github.com/orientechnologies/orientdb-docker.git
-GitCommit: 1cf7837740ef309efc58759b0a33a3fe8c2c5278
-Directory: release/3.2.x/3.2.26-tp3
+GitCommit: 7c3a945a49494db55de300c0193da8d9dd0f51c1
+Directory: release/3.2.x/3.2.27-tp3
 
 Tags: 3.1.20, 3.1
```

Comment:
Any particular reason to keep all these old versions listed here? (Are they still actively maintained?)

Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)